ansible/README.md

```diff
@@ -1,11 +1,32 @@
 # Ansible
 
 This directory contains all [Ansible](http://docs.ansible.com/) playbooks and
-files to setup, configure and deploy services automatically.
+files required to setup services automatically.
+
+## Installing Ansible
+
+If you still do not have Ansible installed in your system, setup EPEL yum
+repository and run:
+
+`yum install -y ansible`
 
 ## Running Ansible playbooks
 
-You can run Ansible playbooks by executing `ansible-playbook` command:
+First, update the host in the hosts.ini section which corresponds to the
+playbook which will be executed (eg if you will execute the jenkins-master
+playbook, update the host in [jenkins-master] section in hosts.ini)
```

Comment:
One may understand that only the host(s) that will be configured should be filled at execution time. I'd say something like: "update all the hosts in the hosts.ini file, paying attention to the groups they belong to. When executing a playbook, only the hosts in the corresponding group will be updated."